packages/shared/services/loc/loc.ts

```diff
@@ -46,13 +44,15 @@ export function displayDate(date: Date) {
 export function displayDateTime(date: Date) {
   try {
     if (isTest) {
-      return moment(date)
-        .utc()
-        .format(cfg.dateTimeFormat);
+      return format(dateToUtc(date), cfg.dateTimeFormat);
     }
-    return moment(date).format(cfg.dateTimeFormat);
+    return format(date, cfg.dateTimeFormat);
   } catch (err) {
     logger.error('displayDateTime()', err);
     return 'undefined';
   }
 }
+
+function dateToUtc(date: Date) {
+  return new Date(date.getTime() + date.getTimezoneOffset() * 60 * 1000);
+}
```

Comment:
hm, curious what is `60 * 1000` for?

Reply:
You can pass the ms since Epoch to the constructor of `Date` to get a new `Date` instance at a certain time.

In this case, `getTime` returns the time in ms since Epoch but `getTimezoneOffset` returns the time in minutes. So to convert minutes to ms you multiply it by 60,000. 

```typescript
const date = new Date()
> undefined
new Date(date.getTime() + date.getTimezoneOffset() * 60 * 1000);
> Date Tue Jan 25 2022 21:55:29 GMT-0600 (Central Standard Time)
```

Reply:
I break it up into `* 60 * 1000` instead of just `* 60000` because that makes it a bit more clear that we're converting to seconds and then to milliseconds.